### PR TITLE
(StorybookUtils): LiveCodeExample - set overflow to visible

### PR DESCRIPTION
**Pull request by mosheKerbel**
@argshook @shlomitc 

before:
Screen Shot 2020-10-26 at 9 19 57

after:
Screen Shot 2020-10-26 at 9 19 41

### CODE REVIEW

**Comment by shlomitc** [comment]
The reason for the overflow is to make the full-screen playground (like in WSR) to support scrolling.
In the inline code-snippets, bad things happen.

In WSR I just removed all these dashed code examples in favor of the plain background ones (also updated the generator).
Anyway, I think it's better to have only a single type of inline code examples.

**Comment by argshook** [comment]
> The reason for the overflow is to make the full-screen playground (like in WSR) to support scrolling.
> In the inline code-snippets, bad things happen.

maybe `overflow: auto` should be applied only for playground, and all
the rest of code examples don't specify overflow at all (thus, the
default `overflow: visible` would work)?

**Comment by mosheKerbel** [comment]
> The reason for the overflow is to make the full-screen playground (like in WSR) to support scrolling.
> In the inline code-snippets, bad things happen.
> 
> In WSR I just removed all these dashed code examples in favor of the plain background ones (also updated the generator).
> Anyway, I think it's better to have only a single type of inline code examples.

When we started documenting base-ui, @Zohar-Telor requested the bg-color to be transparent for components, so it's working that way for all pages in base-ui.

myabe we can do what Arijus suggested?

**Comment by shlomitc** [comment]
yes, I'm good with this solution as well.

File: packages/wix-storybook-utils/src/LiveCodeExample/LiveCodeExample.tsx [hunk 1]
```diff
@@ -37,6 +37,7 @@ export interface Props {
   autoRender?: boolean;
   darkBackground?: boolean;
   noBackground?: boolean;
+  isOverflow?: boolean;
```
Comment (argshook) [inline packages/wix-storybook-utils/src/LiveCodeExample/LiveCodeExample.tsx:40]: @mosheKerbel i don't think you need a dedicated prop, just remove `overflow` css property entirely and add it to playground only

**Review by argshook — APPROVED** [review]
:+1:

**Pull request merged**